<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Searchbook</class>
 <widget class="QDialog" name="Searchbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*border-color: rgb(124, 255, 218);*/
/*background-color: qlineargradient(spread:reflect, x1:0, y1:0.488636, x2:1, y2:0.483, stop:0 rgba(71, 94, 142, 255), stop:1 rgba(254, 252, 255, 243));*/
/*background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0.915254, y2:0.108, stop:0 rgba(71, 94, 142, 255), stop:1 rgba(255, 255, 255, 255));*/
/*background-color: qlineargradient(spread:pad, x1:0.0110455, y1:0.443, x2:0.988636, y2:0.449, stop:0.136364 rgba(71, 94, 142, 255), stop:1 rgba(255, 255, 255, 255));*/
/*background-color: qlineargradient(spread:pad, x1:0.0110455, y1:0.443, x2:0.988636, y2:0.449, stop:0.136364 rgba(111, 72, 72, 255), stop:1 rgba(255, 255, 255, 255));
*/
/*gridline-color: rgb(178, 255, 250);*/</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1177</width>
        <height>508</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">
font: 80 12pt &quot;Segoe Script&quot;;
font: 75 12pt &quot;Segoe Script&quot;;
color:rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>Genre</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_genre1">
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;Segoe Script&quot;;
background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="6">
          <widget class="QTableView" name="tableView_1">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">
font: 75 12pt &quot;Segoe Script&quot;;
color:rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_name1">
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;Segoe Script&quot;;
background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">
font: 75 12pt &quot;Segoe Script&quot;;
color:rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>Author</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_aut1">
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;Segoe Script&quot;;
background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_10">
           <property name="styleSheet">
            <string notr="true">
font: 75 12pt &quot;Segoe Script&quot;;
color:rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>Year</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_year1">
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;Segoe Script&quot;;
background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="whatsThis">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe Script'; font-size:12pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Descriptions&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">
font: 75 12pt &quot;Segoe Script&quot;;
color:rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>Descriptions</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Segoe Script</family>
             <pointsize>8</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 75 8pt &quot;Segoe Script&quot;;
color:rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>*Щоб вибрати книжку ,натисність двічі на поле &quot;Name&quot; або &quot;Genre&quot; та на кнопку &quot;Вибрати книгу&quot;</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLineEdit" name="lineEdit1"/>
         </item>
         <item row="6" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="font">
              <font>
               <family>Segoe Script</family>
               <pointsize>8</pointsize>
               <weight>9</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
font: 75 8pt &quot;Segoe Script&quot;;
color:rgb(255, 0, 0);background :rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Перегляд зайнятих книг</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="styleSheet">
              <string notr="true">
font: 75 8pt &quot;Segoe Script&quot;;
color:rgb(255, 0, 0);background :rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Вийти</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="styleSheet">
              <string notr="true">
font: 75 8pt &quot;Segoe Script&quot;;
color:rgb(255, 0, 0);background :rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_search_1">
             <property name="styleSheet">
              <string notr="true">
font: 75 8pt &quot;Segoe Script&quot;;
color:rgb(255, 0, 0);background :rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Пошук</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_take">
             <property name="styleSheet">
              <string notr="true">
font: 75 8pt &quot;Segoe Script&quot;;
color:rgb(255, 0, 0);background :rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Вибрати книгу </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
